guides/v2.1/cloud/live/sens-data-initial.md

```diff
@@ -132,15 +132,15 @@ To verify your configuration changes:
 3. Click **Stores** > Settings > **Configuration** > General > **General**.
 4. In the right pane, expand **Locale Options**.
 
-	Notice several fields cannot be edited, as shown in the following sample. These configuration settings are maintained by `config.local.php`.
+Notice several fields cannot be edited, as shown in the following sample. These configuration settings are maintained by `config.local.php`.
 
-	![Can't edit certain values in the Admin]({{ site.baseurl }}/common/images/cloud_var_not-editable.png){:width="550px"}
+![Can't edit certain values in the Admin]({{ site.baseurl }}/common/images/cloud_var_not-editable.png){:width="550px"}
 5. In the left navigation pane, click Advanced > **Developer**.
 6. In the right pane, expand **Template Settings**, **JavaScript Settings**, and **CSS Settings**.
 
-	Notice several fields cannot be edited, as shown in the following sample. These configuration settings are maintained by `config.local.php`.
+Notice several fields cannot be edited, as shown in the following sample. These configuration settings are maintained by `config.local.php`.
 
-	![Can't edit certain values in the Admin ]({{ site.baseurl }}/common/images/cloud_var_not-editable2.png){:width="550px"}
+![Can't edit certain values in the Admin ]({{ site.baseurl }}/common/images/cloud_var_not-editable2.png){:width="550px"}
```

Comment:
Indent 4 spaces, otherwise it restarts the numbering.

Reply:
All the images and paragraphs within the numbered steps in this file still need to be indented to nest properly when rendered.